src/pepr/operator/crd/generated/package-v1alpha1.ts

```diff
@@ -549,11 +549,15 @@ export interface Sso {
    * Specifies the protocol of the client, either 'openid-connect' or 'saml'
    */
   protocol?: Protocol;
+  /**
+   * Defines whether the client requires a client secret for authentication
+   */
+  publicClient?: boolean;
   /**
    * Valid URI pattern a browser can redirect to after a successful login. Simple wildcards
    * are allowed such as 'https://unicorns.uds.dev/*'
    */
-  redirectUris: string[];
+  redirectUris?: string[];
```

Comment:
I think I would feel better about this if we were only making it optional _if_ standardFlowEnabled is false. Not sure if we have the flexibility to do that with the version of schema we are using, but we could put the logic in the validator?

Reply:
done 1 min ago...

Reply:
You read my mind @Racer159 !